src/styles/_catalog.scss

```diff
@@ -1,22 +1,30 @@
+$card-size: 200px;
```

Comment:
I think it has to be in file _variables.scss.

Reply:
Agree, lets move it to the file `_variables.scss`